controllers/provider-aws/pkg/webhook/controlplane/ensurer.go

```diff
@@ -126,26 +139,24 @@ var (
 			},
 		},
 	}
-	cloudProviderSecretVolume = corev1.Volume{
-		Name: common.CloudProviderSecretName,
-		VolumeSource: corev1.VolumeSource{
-			Secret: &corev1.SecretVolumeSource{
-				// TODO Use constant from github.com/gardener/gardener/pkg/apis/core/v1alpha1 when available
-				// See https://github.com/gardener/gardener/pull/930
-				SecretName: common.CloudProviderSecretName,
-			},
-		},
-	}
 )
 
 func ensureVolumeMounts(c *corev1.Container) {
 	c.VolumeMounts = controlplane.EnsureVolumeMountWithName(c.VolumeMounts, cloudProviderConfigVolumeMount)
-	c.VolumeMounts = controlplane.EnsureVolumeMountWithName(c.VolumeMounts, cloudProviderSecretVolumeMount)
 }
 
 func ensureVolumes(ps *corev1.PodSpec) {
 	ps.Volumes = controlplane.EnsureVolumeWithName(ps.Volumes, cloudProviderConfigVolume)
-	ps.Volumes = controlplane.EnsureVolumeWithName(ps.Volumes, cloudProviderSecretVolume)
```

Comment:
Just to double check: It's removed because it's not needed to be mounted as it's only used in the env as reference, right?

Reply:
Exactly, and the same is true for some of the other providers. But the checksum is needed in both cases.

Reply:
Thanks!